Analysis/include/AnaTuple.h

```diff
@@ -45,8 +45,17 @@ namespace analysis {
     VAR(bool, is_central_es) \
     VAR(ULong64_t, sample_id) \
     VAR(double, weight) /* weight */ \
+    VAR(double, btag_weight) /* btag_weight */ \
     VAR(float, mva_score) /* mva score */ \
     VAR(ULong64_t, evt)  /* event */ \
+    VAR(Int_t, channelId) /* Channel: eTau, muTau or tauTau */ \
+    VAR(Int_t, num_jets) /* num jets */ \
+    VAR(Int_t, num_btag_loose)  \
+    VAR(Int_t, num_btag_medium) \
+    VAR(Int_t, num_btag_tight)   \
+    VAR(bool, is_vbf) \
+    VAR(bool, is_boosted) \
+    VAR(boost::optional<DiscriminatorWP>, vbf_tag) \
```

Comment:
- don't need to store vbf_tag
- Store p4 for the fat jet and its soft drop mass